apps/frontend/src/app/2_molecules/AavePoolRowTitle/AavePoolRowTitle.tsx

```diff
@@ -9,13 +9,15 @@ import {
 import { AssetRenderer } from '../AssetRenderer/AssetRenderer';
 
 type AaveRowTitleProps = AmountRendererProps & {
+  isOpen?: boolean;
   asset: string;
   label?: string;
 };
 
 export const AaveRowTitle: FC<AaveRowTitleProps> = ({
   asset,
   label,
+  isOpen,
```

Comment:
you can define `isOpen = false` by default and then you don't need to set `isOpen || false` each time you call the AaveRowTitle component